src/config/rcxml.c

```diff
@@ -826,6 +826,14 @@ entry(xmlNode *node, char *nodename, char *content)
 		} else {
 			wlr_log(WLR_ERROR, "Invalid value for <resize popupShow />");
 		}
+	} else if (!strcasecmp(nodename, "left.area.tablet")) {
+		rc.tablet.box.x = tablet_get_area_value(content, "left");
+	} else if (!strcasecmp(nodename, "top.area.tablet")) {
+		rc.tablet.box.y = tablet_get_area_value(content, "top");
+	} else if (!strcasecmp(nodename, "width.area.tablet")) {
+		rc.tablet.box.width = tablet_get_area_value(content, "width");
```

Comment:
Just a thought, should we bound check the width/height against tablet->mm_width

Reply:
I'm doing the bounds check when doing the actual transformation in  https://github.com/labwc/labwc/blob/27bba861e85283cc7619e512fd0b300ead0d5257/src/input/tablet.c#L30 and https://github.com/labwc/labwc/blob/27bba861e85283cc7619e512fd0b300ead0d5257/src/input/tablet.c#L35

Reply:
Good point!